stdlib/threading.pyi

```diff
@@ -215,6 +215,12 @@ if sys.version_info >= (3, 8):
     ExceptHookArgs = _ExceptHookArgs
 
 class Timer(Thread):
+    args: Iterable[Any] | None  # undocumented
```

Comment:
`args` and `kwargs` can't be `None`. If you pass in `None` to `__init__`, they get replaced by an empty list or dict.

Reply:
@Akuli oops, my bad - corrected.